### PR TITLE
Feature/update to polkadot v0 9 5

### PR DESCRIPTION
**Pull request by MRamanenkau**

### CODE REVIEW

File: bin/node/runtime/Cargo.toml [hunk 1]
```diff
@@ -83,7 +83,6 @@ pallet-treasury = { version = "3.0.0", default-features = false, path = "../../.
 pallet-utility = { version = "3.0.0", default-features = false, path = "../../../frame/utility" }
 pallet-transaction-payment = { version = "3.0.0", default-features = false, path = "../../../frame/transaction-payment" }
 pallet-transaction-payment-rpc-runtime-api = { version = "3.0.0", default-features = false, path = "../../../frame/transaction-payment/rpc/runtime-api/" }
-pallet-transaction-storage = { version = "3.0.0", default-features = false, path = "../../../frame/transaction-storage" }
```
Comment (MRamanenkau) [inline bin/node/runtime/Cargo.toml]: A pallet for on-chain storage. [Test log link](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#cee512e48b8d4cea8fad90521ce0cf54)

File: bin/node/runtime/Cargo.toml [hunk 2]
```diff
@@ -44,6 +44,7 @@ frame-benchmarking = { version = "3.1.0", default-features = false, path = "../.
 frame-support = { version = "3.0.0", default-features = false, path = "../../../frame/support" }
 frame-system = { version = "3.0.0", default-features = false, path = "../../../frame/system" }
 frame-system-benchmarking = { version = "3.0.0", default-features = false, path = "../../../frame/system/benchmarking", optional = true }
+frame-election-provider-support = { version = "3.0.0", default-features = false, path = "../../../frame/election-provider-support" }
```
Comment (MRamanenkau) [inline bin/node/runtime/Cargo.toml:47]: An improvement for stacking/election emergencies

File: bin/node/runtime/src/lib.rs [hunk 3]
```diff
@@ -381,10 +383,13 @@ parameter_types! {
 	// For weight estimation, we assume that the most locks on an individual account will be 50.
 	// This number may need to be adjusted in the future if this assumption no longer holds true.
 	pub const MaxLocks: u32 = 50;
+	pub const MaxReserves: u32 = 50;
```
Comment (MRamanenkau) [inline bin/node/runtime/src/lib.rs:386]: Reserve balance [fix](https://github.com/paritytech/substrate/issues/7223)

File: client/network/src/protocol/sync.rs [hunk 4]
```diff
@@ -1145,10 +1145,6 @@ impl<B: BlockT> ChainSync<B> {
 					if let Some(peer) = who.and_then(|p| self.peers.get_mut(&p)) {
 						peer.update_common_number(number);
 					}
-
```
Comment (AndreiNavoichyk) [inline client/network/src/protocol/sync.rs:1143]: Why did you remove this?

File: bin/node/runtime/src/lib.rs [hunk 5]
```diff
@@ -223,6 +223,8 @@ impl frame_system::Config for Runtime {
 	type OnSetCode = ();
 }
 
+impl pallet_randomness_collective_flip::Config for Runtime {}
```
Comment (AndreiNavoichyk) [inline bin/node/runtime/src/lib.rs:226]: Why do we need this?

File: client/network/src/protocol/sync.rs (hunk 4 above)
Comment (MRamanenkau) [reply client/network/src/protocol/sync.rs:1143]: It's probably removed during the merge. There is now no diff with Substrate

File: bin/node/runtime/src/lib.rs (hunk 5 above)
Comment (MRamanenkau) [reply bin/node/runtime/src/lib.rs:226]: randomness-collective-flip [was refactored](https://github.com/paritytech/substrate/pull/9061) and accroding to the comment we need to add 
`impl pallet_randomness_collective_flip::Config for Runtime {}`

**Review by AndreiNavoichyk — DISMISSED** [review]

**Review by AndreiNavoichyk — APPROVED** [review]
